src/main/java/net/sf/jabref/bst/VM.java

```diff
@@ -168,11 +179,6 @@ public void execute(BstEntry context) {
                     throw new VMException("Can only compare two integers with <");
                 }
 
```

Comment:
Both o1 and o2 are Integers (otherwise an Exception would have been thrown on line 179), so should be safe to only compare them using `compareTo`.